scripts/enum/mod.lua

```diff
@@ -912,6 +912,12 @@ xi.mod =
 
     STEP_TP_CONSUMED        = 1077, -- Modifies the amount of TP consumed by dancer steps
 
+    MOVE_SPEED_STACKABLE      = 75, -- Gear movement speed penalties, flee bonus, etc.
```

Comment:
since these are all together they can be placed earlier in the list instead of at the end